app/src/Tasks/stepper.js

```diff
@@ -46,34 +69,48 @@ export default function VerticalLinearStepper() {
   };
 
   return (
-    <Box sx={{ maxWidth: 400 }}>
+    <Box sx={{ maxWidth: 650 }}>
       <Stepper activeStep={activeStep} orientation="vertical">
         {steps.map((step, index) => (
           <Step key={step.label}>
             <StepLabel
               optional={
-                index === 2 ? (
+                index === 4 ? (
                   <Typography variant="caption">Last step</Typography>
                 ) : null
               }
             >
               {step.label}
             </StepLabel>
             <StepContent>
-              <Typography>{step.description}</Typography>
-              <Box sx={{ mb: 2 }}>
+              <Typography>
+                {step.description}
+                <Button
+                  variant="outlined"
+                  size="small"
+                  sx={{ mt: 1, mr: 1, mb: 1, ml: 5 }}
+                  onClick={() => {
+                    alert("clicked");
+                  }}
```

Comment:
this certainly looks right syntax-wise. I'll have to pull it down to take a look locally

Reply:
Okay cool ... looking forward to working through this issue with you.